app/plugins/Wallet/js/lifecycle.js

```diff
@@ -91,14 +91,17 @@ function start() {
 	// DEVTOOL: uncomment to bring up devtools on plugin view
 	// IPC.sendToHost('devtools');
 	
-	// Need to check if wallet's unencrypted
-	IPC.sendToHost('api-call', '/wallet', 'on-opened');
+	// Need to check if wallet's unencrypted, keep checking until we get a response
+	updating = setInterval(IPC.sendToHost, 500, 'api-call', '/wallet', 'on-opened');
```

Comment:
I considered that however to me both of these uses were "updating" so it seemed to fit.

I'll move it over to a seperate `starting` variable for more clarity.